### DISCUSSION TITLE
Game message

### DISCUSSION THREAD

**Discussion by Nexxtar** [adamk33n3r/runelite-watchdog#155] [Ideas]
I am currently using Watchdog alot of different things but what I enjoy most currently is the notification when a parasite spawns on nightmare.
I'm tracking it using game Message which works great.

My idea is instead of having the notification being up for a duration it instead ends from another game message such as "You drink a dose of Relicym's balm"

This way makes it even better tracking whenever you need to drink a dose or not.

**Comment by adamk33n3r** [comment]
You can do this with the overlay notification type and the Dismiss Overlay notification
